src/components/03-components/03-buttons/01-button/_button--single.njk

```diff
@@ -0,0 +1,68 @@
+{#
+    Creates a button.
+    link        (string) - use "a" instead of "button". String should be any valid HREF value
+    modifiers   (array)  - USA "secondary,base,outline,inverse,big,unstyled,hover,active,focus"
+    smodifiers  (array)  - SDS "circle,small"
+    disabled    (bool)   - adds disabled attribute 
+    classes     (string) - extra classes
+    ext         (string) - other attributes i.e. ( tabindex="0" onclick="alert" )
```

Comment:
remove ext

Reply:
we need to be able to add things like tabindex and other attributes if needed